README.md

```diff
@@ -165,6 +165,7 @@ $ make -C test/gtest test
     * posix, sysv, [cma](https://dl.acm.org/citation.cfm?id=2616532), [knem](http://knem.gforge.inria.fr/), and [xpmem](https://github.com/hjelmn/xpmem)
 * TCP/IP
 
+  **NOTE:** [Infiniband](https://www.infinibandta.org/) and [RoCE](http://www.roceinitiative.org/) transports requires rdma-core >= 28.0 or MLNX_OFED >= 5.0
```

Comment:
requires --> require

Reply:
ditto

Reply:
Starting from what UCX version ?

Reply:
> Starting from what UCX version ?

I think master version of README should be relevant to master version, otherwise we should describe it for all versions, what can be problematic to restore. @yosefe WDYT?

Reply:
@evgeny-leksikov This is not accurate. The readme is used as a front page of the project. If you look at the prevision notes you will see that we explicitly call out UCX versions with problems and etc. We need to specify UCX version that requires latest MOFED and is prone to this specific problem.

Reply:
@shamisp do you mean Known Issues section? In this PR I would like to highlight a requirement but not an issue.

Reply:
@evgeny-leksikov It is okay. Do we require this for all versions or starting from certain version ?

Reply:
@shamisp basically every SW requires updates time to time, we cannot track every UCX version in past where any issue was fixed. especially if it's found in future. Please note, that actualy this is not an issue, issue was highlited by warning in configure output.
I agree that switching to new MOFED version was not highlited in time, this is an issue and probably should be handled separately.

Reply:
There are minimum software requirement for every SW version.  I think the statement "Use latest MOFED" by itself is not very useful.

Reply:
Why? if you have an issue - try to upgrade and in many cases it's already fixed, If everything is stable and working as expected - juct don't touch it :) IMHO front page of the project is important firstly for newcomers.

Reply:
Well, sometime UCX is behind rdma-core on changes. So the advice "use latest" is not always true. 
My suggestion - just pick a version; UCX 1.12.0 and above require MOFED XX. If later you discover that 1.11.x requires later MOFED, we can update the message.

Reply:
ok, added 1.12.0